core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -64,11 +64,11 @@ import org.apache.spark.util._
  * Main entry point for Spark functionality. A SparkContext represents the connection to a Spark
  * cluster, and can be used to create RDDs, accumulators and broadcast variables on that cluster.
  *
- * Only one SparkContext may be active per JVM.  You must `stop()` the active SparkContext before
- * creating a new one.  This limitation may eventually be removed; see SPARK-2243 for more details.
+ * Only one SparkContext may be active per JVM. You must `stop()` the active SparkContext before
```

Comment:
I still just don't think it's worth removing spaces. They don't even matter in rendered HTML